lib/dish.rb

```diff
@@ -0,0 +1,26 @@
+class Dish
+  attr_reader :name, :description, :price
+  def initialize(dish_hash = {name: "", description: "", price: 0})
+    @name = dish_hash[:name]
+    @description = dish_hash[:description]
+    @price = dish_hash[:price]
+  end
+
+  def set_all(dish_hash)
```

Comment:
When would you use this method?

Reply:
I added this at the time of making the class as I thought i might need it... classic case of YAGNI really, basically forgot to delete it.